Dockerfile

```diff
@@ -1,45 +1,38 @@
-FROM ruby:2.3
+FROM ruby:2.3-alpine
 
-RUN apt-get update -qq && \
-    apt-get install -qq -y nginx && \
-    # temporay for debugging remove nano
-    apt-get install -qq -y nano && \
-    apt-get install -qq -y supervisor && \
-    apt-get autoremove -y && \
-    apt-get clean -y && rm -rf /var/lib/apt/lists/*
+MAINTAINER SLAPI Dev Team
 
-# Setup App Environment and User
-ENV APP_HOME /slapi
+ENV APP_HOME /usr/src/slapi
 
-RUN mkdir $APP_HOME && \
-    adduser slapi --disabled-password --gecos "" && \
-    chown -R slapi:slapi $APP_HOME && \
-    echo "slapi            ALL = (ALL) NOPASSWD: ALL" >> /etc/sudoers
-
-RUN mkdir $APP_HOME/tmp && \
-    mkdir $APP_HOME/tmp/sockets && \
-    mkdir $APP_HOME/tmp/pids && \
-    mkdir $APP_HOME/log
+RUN mkdir -p $APP_HOME &&\
+    mkdir -p $APP_HOME/log &&\
+    chmod -R 777 $APP_HOME &&\
+    chmod -R 777 /usr/local/lib/ruby/
```

Comment:
So, I can probably not chmod anymore. When I was running separate user in the container instead of being root I had to make tons of file changes to get access.

When you do COPY or ADD for a project, it copies everything into the container as root which is crazy annoying.

The ruby one at least may have to stay, I keep running into to issues where root does not have write access into `/usr/local` which is odd.